src/login/login.component.test.tsx

```diff
@@ -50,7 +58,7 @@ describe(`<Login />`, () => {
     expect(performLogin).toHaveBeenCalled();
   });
 
-  it(`send the user to the location select page on login`, () => {
+  xit(`send the user to the location select page on login if there is more than one location`, () => {
```

Comment:
The `historyMock` is colliding between this test and the new one. With this test disabled, everything works. When this test is enabled, `historyMock.push` in the new test seems to receive the value it has in this test.

Reply:
take a look at how I mocked this in [patient search](https://github.com/openmrs/openmrs-esm-home/blob/master/src/helpers/render-with-router.tsx), might help out.

Reply:
Thanks @fatmali ! Opened https://github.com/openmrs/openmrs-esm-login/pull/40

Reply:
@brandones should we close this as #40 has been merged?

Reply:
@fatmali No, now that #40 has been merged, I can continue working on this one. Thank you for your support :)